### DISCUSSION TITLE
chatwoot-base container exits with only "switch to inspect mode" in logs

### DISCUSSION THREAD

**Discussion by etranger7** [chatwoot/chatwoot#6338] [General]
**Describe the bug**

When setting up chatwoot using docker, the base container exits and doesn't restart.
I don't know if this is actually a bug or if this is exactly what it's supposed to do.

**To Reproduce**

1. set variables  in .env 
2. docker compose up
3. progress shows "switch to inspect mode" "chatwoot-base exited with code 0"
4. See the error

**Expected behavior**

I'm not sure what this container is supposed to do, whether it's supposed to exit  like this.

![Screenshot 2023-01-23 120742](https://user-images.githubusercontent.com/5760298/214139430-7b5ac43e-662c-4a0d-bb32-fce9545c5925.jpg)

**Environment**

Self-hosted Docker on Debian 11

**Docker** (If applicable)

Please share the output of the following.
- docker version: 20.10.23
- docker-compose version: 1.29.2

**Additional context**

Here is another issue where it's mentioned that the base is there to build the other containers. 
https://github.com/chatwoot/chatwoot/issues/3234
So is it supposed to be removed afterwards and it's not getting removed?

I tried this on 2 different machines. Same .env file.

**Comment by vishnu-narayanan** [comment]
Are you facing any issues? `base` should exit with `0`.

Could you share the output of `docker ps`?

**Reply by etranger7** [reply]
Hi, thank you for looking into this.
I didn't realize this is the intended behavior and the normal operation.
Thank you for confirming it.
I'm not experiencing any issues so far.
Thanks for confirming.
